src/components/settings/SettingsLMSTab/constants.js

```diff
@@ -0,0 +1,6 @@
+export const BLACKBOARD_ICON = 'https://uthsc.edu/tlc/images/bblogoc.png';
```

Comment:
I know this way is probably not advised, but couldn't really find any good examples of best practice of other's icons!